src/renderer/containers/GlobeView.tsx

```diff
@@ -192,9 +194,9 @@ class GlobeView extends React.Component<IGlobeViewProps & IGlobeViewOwnProps & D
         const baseDir = this.props.workspace.baseDir;
         const url = getTileUrl(this.props.baseUrl, baseDir, layer);
         let rectangle = Cesium.Rectangle.MAX_VALUE;
-        if (imageLayout.sector) {
-            const sector = imageLayout.sector;
-            rectangle = Cesium.Rectangle.fromDegrees(sector.west, sector.south, sector.east, sector.north);
+        if (imageLayout.extent) {
+            const extent = imageLayout.extent;
+            rectangle = Cesium.Rectangle.fromDegrees(extent.east, extent.south, extent.west, extent.north);
```

Comment:
did you switch the arguments by accident or by intent?

Because the order is (west, south, east, north)
https://cesiumjs.org/Cesium/Build/Documentation/Rectangle.html#.fromDegrees